fec/fec/static/js/calendar.js

```diff
@@ -47,27 +85,48 @@ Calendar.prototype.updateLinks = function(params) {
   };
   this.$download.html(templates.download(urls));
   this.$subscribe.html(templates.subscribe(urls));
+
+  new dropdown.Dropdown(this.$download, {checkboxes: false});
```

Comment:
It looks like we're instantiating a new pair of `Dropdown` objects on each filter. That means we're going to continue piling up event listeners for earlier dropdowns. We may want to add a `destroy` method to dropdowns and invoke it here to avoid listener clutter.